### DISCUSSION TITLE
Prettier / VSCode: Wrap tailwind classes

### DISCUSSION THREAD

**Discussion by ciarantr** [tailwindlabs/tailwindcss#9003] [Help]
Hi, is there a way to wrap each class on its own line in vscode using prettier extention?

![code](https://user-images.githubusercontent.com/58338331/182123803-7d43b498-b924-410f-8ea6-672dc303aef1.png)

**Comment by yomajo** [comment]
Just started using tailwind. Using flask with tailwind cli. node is not an option. Same problem. Default vscode formatter does not wrap. Out of the box prettier - does not wrap.
